include/quic/btstream.hpp

```diff
@@ -224,19 +239,33 @@ namespace oxen::quic
 
         void closed(uint64_t app_code) override;
 
+        /// Registers an individual command to be recognized by this BTRequestStream object.  Can be
+        /// called multiple times to set up multiple commands.  As an alternative (or supplement)
+        /// you can also specify a single handler during construction to be called for any invoked
+        /// commands that are not created via `register_command` calls.
         void register_command(std::string endpoint, std::function<void(message)>);
 
         const Address& local() const { return conn.local(); }
 
         const Address& remote() const { return conn.remote(); }
 
       private:
+        // Optional constructor argument: stream close callback
         void handle_bp_opt(std::function<void(Stream&, uint64_t)> close_cb)
         {
             log::debug(bp_cat, "Bparser set user-provided close callback!");
             close_callback = std::move(close_cb);
         }
 
+        // Optional constructor argument: generic request handler.  If set, this is invoked for all
+        // incoming requests to endpoints that do not have a `register_command`.  (And so if
+        // `register_command` is not used at all, this can handle all endpoint).
+        void handle_bp_opt(std::function<void(oxen::quic::message m)> request_handler)
```

Comment:
Do you think it would be better to use a member function `register_generic_handler`, similar to how we use `register_command` for each specific endpoint?

I don't feel too strongly either way, but the explicit-ness of the method feels like a convincing idea to me. 

If you didn't want to create a new method, we could even change the parameters of  `register_command` like:
```c++
void BTRequestStream::register_command(std::optional<std::string> ep, std::function<void(message)> func)
{
    endpoint.call(
        [this, ep = std::move(ep), func = std::move(func)]() mutable { 
            if (ep)
                func_map[std::move(ep)] = std::move(func);
            else
                generic_handler = std::move(func);
        });
}
```
So setting the generic handler means passing `std::nullopt` to the first argument.

Reply:
I suppose it could have a method for it as well, but it is pretty simple to be able to just slap it into the constructor.

Reply:
There's a second reason for it being useful in the constructor: we don't have to worry about thread synchronization during construction and so can just set it directly, where the method version (which I'm also adding) has to worry about this by deferring the assignment to an event loop call.